docs/book/user-guide/production-guide/remote-storage.md

```diff
@@ -0,0 +1,165 @@
+---
+description: Transitioning to remote artifact storage.
+---
+
+# Chapter 3: Connecting Remote Storage
+
+In the previous chapters, we've been working with artifacts stored locally on our machines. This setup is fine for individual experiments, but as we move towards a collaborative and production-ready environment, we need a solution that is more robust, shareable, and scalable. Enter remote storage!
+
+Remote storage allows us to store our artifacts in the cloud, which means they're accessible from anywhere and by anyone with the right permissions. This is essential for team collaboration and for managing the larger datasets and models that come with production workloads.
+
+When using a stack with remote storage, nothing changes except the fact that the artifacts
+get materialized in a central, remote, storage location. This diagram explains the flow:
+
+<figure><img src="../../.gitbook/assets/local_run_with_cloud_artifact_store.png" alt=""><figcaption><p>Sequence of events that happen when running a pipeline on a remote artifact store.</p></figcaption></figure>
```

Comment:
For some reason, this image is not rendering anymore on Gitbook spaces for me.

Reply:
fixed